IPNI.md

```diff
@@ -292,6 +292,11 @@ Specified protocols are expected to be ordered in increasing order.
 * http
     * the proposed `uvarint` protocol is `0x3D0000`.
     * the following bytes are not yet defined.
+* Named Record
+    * Protocol `uvarint` is <TBD> in the multicodec table
+    * the following bytes should be a dag-cbor encoded struct of:
```

Comment:
Thoughts on if we should be saving on bytes here by making this a CBOR tuple where element 0 is the name and 1 is the Record? I took this format ro start with to mimic the GraphSync-FILv1 one.